{"author":{"id":"4d85ccdde3ad47c902de21b580354b67ed0cb07caa04e32d04ea93112f7f027e"},"ops":[{"type":3,"timestamp":1723734774,"nonce":"V8/UExxWeNsYUwSJTcxHSJXHx4Y=","metadata":{"github-id":"IC_kwDOCGKeh86IlXcA","github-url":"https://github.com/git-bug/git-bug/issues/1014#issuecomment-2291496704"},"message":"I think the confusion here stems from the error message telling the user the PID of the process that owns the lock file (which can become stale or inaccurate). Checking for a process by name is a difficult problem to solve for accurately.\n\nI think the right thing to do here would be to change the approach and altogether avoid trying to be smart: if a lock file exists, error out (optionally, we could prompt the user and ask if they want to remove it, or otherwise add a `git bug unlock` command to do that for them).","files":null}]}